### PR TITLE
Documentation: Figures for nonlinear stokes solve

### PR DESCRIPTION
**Pull request by ljhwang**
Cleans up some of the figure captions for the and adds a missing circle. Please feel free to correct and make your own changes especially if I placed the circle wrong. This is my first foray into tikz!

### CODE REVIEW

File: doc/sphinx/user/methods/nonlinear-stokes-solvers.md [hunk 1]
```diff
@@ -264,6 +264,7 @@ Now that we have a general understanding of how the Newton solver works, let's a
     \draw [blue, thick, ->] (0.539987,0.5) -- (0.539987,0.475140);
     \draw [red, thick] (0.539987,0.475140) -- (0.576478,0.5);
     \draw [red, thick, ->] (0.576478,0.5) -- (0.576478,0.499433);
+    \draw[red] (576478,0.499433) circle (0.05);
```
Comment (ljhwang) [inline doc/sphinx/user/methods/nonlinear-stokes-solvers.md:267]:
```suggestion
    \draw[red] (0.576478,0.499433) circle (0.05);
```

**Comment by bangerth** [comment]
This has acquired a merge conflict. Do you know how to address it?

**Comment by ljhwang** [comment]
Ahh, I created a conflict with myself.  Neither of the pull requests is correct.  I think the best thing is to accept the first (longer) figure caption and then perhaps I go back and fix it - again.

**Review by gassmoeller — APPROVED** [review]
:+1:

**Pull request merged**